tests/test_harvest_bookmarks.py

```diff
@@ -430,199 +425,147 @@ def do_test(self, conn_id):
 
         for stream, expect in self._teardown_delete.items():
             if expect:
-                logging.info("last synced {}: {}".format(stream, expect[-1]))
+                LOGGER.info("last synced {}: {}".format(stream, expect[-1]))
 
         try:
-            logging.info("Inserting roles")
+            LOGGER.info("Inserting roles")
```

Comment:
Create one utils file and move this code there, if possible create on function to mitigate duplication.

Reply:
Moved this code to common function in `utils.py`